src/js/utils/option-utils.jsx

```diff
@@ -136,8 +136,8 @@ export const debounceProductsInOrders = (waitTime, minSearchLength, vendor, dest
             name: obj.name,
             productCode: obj.productCode,
             handlingIcons: obj.handlingIcons,
-            label: `${obj.productCode} - ${obj.translatedName ?? obj.name}`,
-            hasTranslatedName: !!obj.translatedName,
+            label: `${obj.productCode} - ${obj.displayNames?.default ?? obj.name}`,
+            hasTranslatedName: !!obj.displayNames?.default,
```

Comment:
same as above:
```suggestion
           displayName: obj.displayNames?.default,
```

Reply:
From what I can see, it appears to be that `hasTranslatedName` from **debounceAvailableItemsFetch** and **debounceProductsInOrders** is not being used anywhere, so I think it's safe to remove it.